pkg/storage/queue.go

```diff
@@ -531,6 +531,10 @@ func (bq *baseQueue) processReplica(
 				log.VEventf(queueCtx, 3, "not holding lease; skipping")
 				return nil
 			}
+			if _, harmless := err.GetDetail().(*roachpb.RangeNotFoundError); harmless {
+				log.Event(queueCtx, "range not found on node; skipping")
```

Comment:
I think you can merge this error with the one above and use
log.VEventf(queueCtx, 3, "%s; skipping", err)